torchvision/prototype/transforms/_geometry.py

```diff
@@ -20,9 +20,25 @@ def bounding_box(input: BoundingBox) -> BoundingBox:
         x = input.image_size[1] - (x + w)
         return BoundingBox.from_parts(x, y, w, h, like=input, format="xywh").convert(input.format)
 
+    @staticmethod
+    def keypoint(input: Keypoint) -> Keypoint:
+        x, y = input.unbind(-1)
+        _, width = input.image_size
+        x = width - x
+        coordinates = torch.stack((x, y), dim=-1)
+
+        idcs = list(range(input.num_keypoints))
+        for symmetry, first, second in input.symmetries:
+            if symmetry != KeypointSymmetry.VERTICAL:
+                continue
+
+            idcs[first], idcs[second] = second, first
+
+        return Keypoint(coordinates[..., idcs, :], like=input)
+
 
 class Resize(Transform):
-    NO_OP_FEATURE_TYPES = {Label}
+    NO_OP_FEATURE_TYPES = {Label, Keypoint}
```

Comment:
Ok if we leave this as a TODO, but we should change the coordinates of the keypoints following the rescaling factor

Reply:
Yes, in general all geometric transforms need to support `Keypoint`'s. We can probably share a lot of functionality with the bounding box kernels.